verification.py

```diff
@@ -1641,6 +1641,60 @@ def test_sojourn_time2(self):
         pyplot.savefig(f, dpi=72)
         pyplot.close("all")
 
+    def test_sojourn_time3(self):
+        """
+        testing against expected sojourn time of a
+        neutral mutation over a range of population
+        sizes
+        """
+        refsizes = np.linspace(1e2, 1e4, 10)
+        nreps = 50
+        seqlen = 1e4
+        dt = 1e-6
+        pos = np.floor(seqlen / 2)
+        df = pd.DataFrame()
+        data = collections.defaultdict(list)
+        for n in refsizes:
+            s = 0
+            p0 = 1.0 / (2 * n)
+            p1 = 1 - p0
+            mod = msprime.SweepGenicSelection(
+                start_frequency=p0, end_frequency=p1, s=s, dt=dt, position=pos
+            )
+            replicates = msprime.sim_ancestry(
+                5,
+                population_size=n,
+                model=mod,
+                sequence_length=seqlen,
+                num_labels=2,
+                num_replicates=nreps,
+            )
+
+            reptimes = np.zeros(nreps)
+            i = 0
+            for x in replicates:
+                tree_times = np.zeros(x.num_trees)
+                j = 0
+                for tree in x.trees():
+                    tree_times[j] = np.max([tree.time(root) for root in tree.roots])
+                    j += 1
+                reptimes[i] = np.max(tree_times)
+                i += 1
+            data["alpha_means"].append(np.mean(reptimes))
+            data["exp_means"].append(2 * n)
```

Comment:
This is why comments are useful: many will think that the expected time to fixation is 4N, but here it is written as 2N--why?